### DISCUSSION TITLE
어떻게 하면 멀티서버의 애니메이션 동기화를 할 수 있을까요?

### DISCUSSION THREAD

**Discussion by ted002** [naverz/zepeto-studio-kor#627] [5. Q&A]
현재 V2버전에 들어오고나서 부터 기본 애니메이션이 매우 다채로워졌습니다

허나 제공해주신 멀티서버 예제로는 움직임과 점프정도만 따라할뿐, 자연스러운 회전, 스탬핑등은 적용되지 않습니다

또한 자체적으로 만든 애니메이션, 혹은 믹사모의 애니메이션을 적용하는것도 쉽지 않습니다.

원래는 애니메이터에서 current 애니메이션의 isName을 활용해서 애니메이션에 따라 재생하는것도 시험해 보았는데요, 이러한 방법으로는 어색하게 따라하거나, A기기에서는 회전 애니메이션을 재생하는데 B기기에서는 재생하지 않는 등의 문제가 생겼습니다

혹시 효과적으로 애니메이션을 동기화 하는 법을 알고계시다면 공유 가능할까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 
https://github.com/naverz/zepeto-studio-kor/discussions/645
와 동일한 이슈로 답변 참고 부탁 드립니다.
